db/migrate/20210803162052_migrate_thm_csr_variant.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+# RAILS_ENV=production bundle exec rails db:migrate:up VERSION="20210803162052"
+# rubocop:disable Metrics/CyclomaticComplexity
+class MigrateThmCsrVariant < Mongoid::Migration
+  def self.up
+    @logger = Logger.new("#{Rails.root}/log/migrate_thm_csr_variant.log") unless Rails.env.test?
+    @logger.info "Script Start - #{TimeKeeper.datetime_of_record}" unless Rails.env.test?
+    people = Person.all
+    people.each do |person|
```

Comment:
Can loop through the family and also will result in less complexity for the report.